src/Raven.Server/Documents/Indexes/Workers/HandleReferences.cs

```diff
@@ -354,6 +360,16 @@ bool CanContinueReferenceBatch()
                         _referencesState.Clear(earlyExit, actionType, collection, indexContext);
                     }
                 }
+                }
```

Comment:
Formatting issue. I know that one you fix it you'll have ~200 lines changed but when you put it in a dedicated commit I think that will be fine

Reply:
What do you think? Someone will come here one day and do the formatting anywya

Reply:
done